tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -140,6 +142,47 @@ function get_container_runtime() {
 	fi
 }
 
+function is_containerd_capable_of_using_drop_in_files() {
+	local runtime="$1"
+
+	if [ "$runtime" == "crio" ]; then
+		echo "false" 
```

Comment:
Did you consider returning `0` for true and `1` for false here, so avoid needing to capture the `echo` output? It's not a big deal, as it's a pretty self-contained function, so we are unlikely to need to do debug, or call a chattier function in here.

Reply:
I did, and I will change that back to used 0 / 1.